magenta/models/shared/melody_rnn_generate.py

```diff
@@ -125,13 +126,21 @@ def run_with_flags(melody_rnn_sequence_generator):
   if not os.path.exists(FLAGS.output_dir):
     os.makedirs(FLAGS.output_dir)
 
+
+#primer_sequence from written melody or given midi file or random note between midi #48 = 4C to midi #84 = 7C
```

Comment:
@cghawthorne and @danabo  So I changed my code to just reset primer_sequence to ['60'] if no primer_midi or primer_melody. 

https://github.com/hpssjellis/magenta/blob/master/magenta/models/shared/melody_rnn_generate.py#L139-L142

That should be a less confusing method to set the primer_sequence while allowing the original code to change it to a random first note. I will test my code. Perhaps you guys have a better way.